src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.scss

```diff
@@ -1 +1,30 @@
+.tooltip {
+  display: flex;
+  flex-direction: row;
+  border-radius: 5px;
+  background: var(--rundown-scrollbar-color);
+  height: 110px;
+
+  .header {
+    padding: 3px;
+    border-top-right-radius: 5px;
+    border-bottom: solid 1px var(--black-color);
+
+    .label {
+      margin-left: 5px;
+      color: var(--white-color);
+      font-weight: bolder;
```

Comment:
`font-weight: bolder` doesn't look nice on windows machines:
![image](https://github.com/tv2/ng-sofie/assets/23049647/3b8620b7-9667-45a7-a9d4-fa7abf5428f0)

Reply:
I removed the bold font weight since the color should be enough to differ between the fields.